grammar/mongo.g4

```diff
@@ -57,7 +57,8 @@ propertyName: StringLiteral | IDENTIFIER;
 
 comment: SingleLineComment | MultiLineComment;
 
-RegexLiteral: '/' (~[/] | '\\/')+ '/' (RegexFlag)*;
+RegexLiteral:
+	'/' (~[/\n*] | '\\/') (~[/\n] | '\\/')* '/' (RegexFlag)*;
```

Comment:
Don't you need \r, too?

Reply:
I would add a comment explaining what you're doing.

Reply:
is // allowed in the Console as a regular expression?

Reply:
// is not allowed.

Reply:
It looks like \t is allowed, so I'm going to allow that.